lighthouse-core/lib/lh-error.js

```diff
@@ -22,7 +22,7 @@ const UIStrings = {
   /** Error message explaining that Lighthouse could not load the requested URL and the steps that might be taken to fix the unreliability. */
   pageLoadFailedWithDetails: 'Lighthouse was unable to reliably load the page you requested. Make sure you are testing the correct URL and that the server is properly responding to all requests. (Details: {errorDetails})',
   /** Error message explaining that the credentials included in the Lighthouse run were invalid, so the URL cannot be accessed. securityMessages will be replaced with one or more strings from the browser explaining what was insecure about the page load. */
-  pageLoadFailedInsecure: 'The URL you have provided does not have valid security credentials. {securityMessages}',
```

Comment:
revert?

Reply:
Oh @patrickhulce was suggesting this in an obve comment
> Reword 'INSECURE_DOCUMENT_REQUEST' to say 'valid SSL certificate'?
https://github.com/GoogleChrome/lighthouse/pull/8294#discussion_r275837518

Reply:
it seemed like we never really touch these strings and this was the closest PR we'd naturally get :)

Reply:
The other changes we're making here that touched the triggering of these is what originally brought this up, but I'm fine to punt if others feel it's sufficiently unrelated.

Reply:
I'd say we probably shouldn't have it in this PR, but we could probably get everybody who needs to weigh in if we really want to :)

It's a translated string, is used as a return value for the PSI API if it runs into an SSL error, etc, so we'd definitely want to get eyes on it before landing it

Reply:
aye, sorry @wardpeet I'll open a different PR for this then :)

Reply:
wait hang on, I never actually wanted this string to be changed in lh-error in this PR, lol

I was just talking about the one in the detection popup :)

I'll still file a different PR, but let's not revert the extension-specific string?

Reply:
fixed, no problem :)